src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -84,6 +84,14 @@ public function getConfigTreeBuilder()
                                 'base_urls' => array_values(array_unique(array_merge($config['base_urls']['http'], $config['base_urls']['ssl']))),
                             );
                         }
+                    } else {
```

Comment:
Right. That's why I initially added it in a separate section in the first place.